### PR TITLE
Update with auto reload issue

### PR DESCRIPTION
**Pull request by robeerob**

### CODE REVIEW

**Comment by Rafase282** [comment]
# :+1: LGTM

File: Known-Issues-With-Codepen.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Free Code Camp learners are encouraged to use [Codepen.io](http://www.codepen.io
 1. **URL issue with anchor tags:** CodePen overrides `<a href='URL'>` anchor elements, which means that you have to add a `target='_blank'` to your anchor elements - otherwise they won't work.
 2. **https plugin issue:** The Chrome extension "HTTPS Everywhere" (created by the [Electronic Frontiers Foundation](http://www.eff.org/)) has been known to interfere with AJAX calls. Because the extension automatically uses HTTPS, it can cause a "mixed content" error that prevents the JSON/XML data from being loaded. If you encounter such an error, make sure that your plugins are not the culprit. Optionally you could use [crossorigin.me](http://crossorigin.me) as a proxy.
 3. **imgur hotlinking:** If you use images from [http://imgur.com](http://imgur.com) they will not show up most of the time, this is due to their TOS. A way to solve this is to use an alternate service like [http://postimg.org](http://postimg.org)
+4. **auto reload:** By default, everytime you make changes in the HTML or JS editor windows, the preview window refreshes. You can turn this off and enable a "Run Button", by going to Settings > Behaviour > "Want a Run Button?" and unticking the box.
```
Comment (abhisekp) [inline Known-Issues-With-Codepen.md]: Might want to add an alternative i.e. http://jsbin.com which has CSS code injection with hot reloading.

**Comment by robeerob** [comment]
@abhisekp While I do like jsbin, it's not really an option to use for the ziplines that require codepen for submission.

**Comment by abhisekp** [comment]
@robeerob I used Plunker for my Ziplines and submitted them. One can use any site for their Ziplines. Anyway, adding alternatives might help others. So I was thinking it would be nice to add other alternatives.  
Let the [**Label: WORK NEEDED: Implement Suggestions**](https://github.com/FreeCodeCamp/wiki/labels/Work Needed: Implement Suggestions) be present so that someone could work on this.

**Comment by robeerob** [comment]
@abhisekp I went and checked the ["Get Set for our Front End Development Projects"](https://www.freecodecamp.com/challenges/get-set-for-our-front-end-development-projects) challenge, and nowhere does it mention that using CodePen is optional. Also, on the individual zipline pages, it explicitly says to use CodePen:

> When you are finished, click the "I've completed this challenge" button and include a link to your CodePen.

I would therefore prefer if we get @QuincyLarson or @BerkeleyTrue to weigh in on this before we start recommending any other tools for the ziplines.

**Pull request merged**